custom/dhis2/payload_generators.py

```diff
@@ -45,6 +45,15 @@ def enabled_for_domain(domain):
     def get_headers(self, repeat_record, payload_doc):
         return {'Content-type': 'application/json'}
 
+    def _update_instance(self, dhis2_api, case):
+            instance = dhis2_api.get_te_inst(case['external_id'])
+            instance.update({dhis2_attr: case[cchq_attr]
+                             for cchq_attr, dhis2_attr in NUTRITION_ASSESSMENT_PROGRAM_FIELDS.iteritems()
+                             if getattr(case, cchq_attr, None)})
+            if 'Gender' in instance:
+                instance['Gender'] = instance['Gender'].capitalize()
```

Comment:
The wvlanka app may reset the gender to "male" or "female", and the update will fail if it isn't capitalized as "Male" or "Female".